### PR TITLE
Gatsby: Add content translations from Drupal.

### PR DESCRIPTION
**Pull request by miromichalicka**
```
query MyQuery {
  allArea(filter: {langcode: {eq: "en"}}) {
    nodes{
      id
      name
    }
  }
}
```

this seems to be working. We probably need to test it on multiple different queries.

### CODE REVIEW

File: gatsby/config/gatsby-config.ts [hunk 1]
```diff
@@ -3,8 +3,25 @@ const config = {
...
       options: {
-        baseUrl: 'https://covid.pristup.net/',
+        baseUrl: 'https://covid.pristup.net',
         apiBase: 'api',
+        translation: true,
+        languageConfig: {
+          defaultLanguage: 'cs',
+          enabledLanguages: ['en', 'cs'],
+          translatableEntities: [
+            { type: 'taxonomy_term', bundle: 'area', id: 'area' },
```
Comment (miromichalicka) [inline gatsby/config/gatsby-config.ts:13]: @kanei could you please check, if I included all translatable content?

**Review by fidelman — APPROVED** [review]

File: gatsby/gatsby-source-drupal/gatsby-node.js [hunk 2]
```diff
@@ -0,0 +1,443 @@
...
+        console.info(`This Drupal data source is configured as multilingual.`);
+        for (let i = 0; i < languageConfig.enabledLanguages.length; i++) {
+          let currentLanguage = languageConfig.enabledLanguages[i];
+          let dataForLanguage;
+
+          if (currentLanguage === languageConfig.defaultLanguage) {
+            dataForLanguage = await getNext(url);
+          } else {
+            var baseUrlWithoutTrailingSlash = baseUrl.replace(/\/$/, ``);
+            var urlPath = url.href.replace(`${baseUrlWithoutTrailingSlash}/${apiBase}/`, "");
```
Comment (miromichalicka) [inline gatsby/gatsby-source-drupal/gatsby-node.js:308]: Is there no better JS URL parsing?

**Comment by miromichalicka** [comment]
Screenshot from 2020-11-05 02-13-31
And I can confirm that there is CS and EN version of the same resource, so it looks promising.

**Comment by miromichalicka** [comment]
Just one note: I had to run this before I was able to successfully download all content from Drupal ```echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p``` It's some OS property, no idea...

**Pull request merged**